src/Model/Response/ReferenceData.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\CdtClient\Model\Response;
+
+/**
+ * Class ReferenceData.
+ *
+ * Represents the business reference data obtained from the CDT API.
+ * Stores the list of departments, priorities, purposes, delivery modes,
+ * confidentialities, languages, statuses, services and send options.
+ * It also contains the contact data for the translation service.
+ */
+class ReferenceData
+{
+    /**
+     * The list of departments (code and description).
+     * Ex. 250772 / Service Finance
+     *
+     * @var ReferenceItem[]
```

Comment:
this is already good, but I would like to enforce the typecast more, lets have a ReferenceDataColletion class, iterable here instead of an array (and where enywhere else it makes sense